### PR TITLE
tests/network-(bridge-firewall|ovn): add IPv6 CIDR size checks tests

### PR DESCRIPTION
**Pull request by mihalicyn**
tests for https://github.com/canonical/lxd/pull/14276

### CODE REVIEW

File: tests/network-bridge-firewall [hunk 1]
```diff
@@ -18,6 +18,17 @@ lxc network create lxdbr0 \
     ipv4.address=192.0.2.1/24 \
     ipv6.address=2001:db8::1/64 \
     ipv4.dhcp.ranges=192.0.2.2-192.0.2.199
+
+echo "=> Ensure that IPv6 CIDR size checks in LXD are correct"
+# Check that we produce a warning if user sets a prefix larger than /64
+lxc network set lxdbr0 ipv6.address=2001:db8::1/48
+lxc warning list | grep "IPv6 networks with a prefix larger than 64"
```
Comment (simondeziel) [inline tests/network-bridge-firewall:25]: A `lxc warning delete --all` is probably needed here.

File: tests/network-bridge-firewall [hunk 2]
```diff
@@ -18,6 +18,17 @@ lxc network create lxdbr0 \
...
+# Check that there is no warning if user sets prefix smaller than /64
+lxc network set lxdbr0 ipv6.address=2001:db8::1/65
+! lxc warning list | grep "IPv6 networks with a prefix larger than 64"
```
[the remark this answers is not in the document]
Comment (mihalicyn) [reply tests/network-bridge-firewall]: Thanks, fixed!

File: tests/network-bridge-firewall (hunk 1 above)
Comment (mihalicyn) [reply tests/network-bridge-firewall:25]: Nice catch!
But actually, as far as I understand we do cleanup old warnings automatically when configuration issue is resolved:
https://github.com/canonical/lxd/blob/6c31513b084098f3891ee9a9312c1d6f8a1380e8/lxd/network/driver_bridge.go#L1518

**Review by tomponline — APPROVED** [review]

File: tests/network-bridge-firewall (hunk 1 above)
Comment (simondeziel) [reply tests/network-bridge-firewall]: Oh, I didn't know about this, thanks!

**Pull request merged**